prisma/schema.prisma

```diff
@@ -33,45 +33,36 @@ model Account {
 
 model User {
   id            String    @id @default(cuid())
+  archived      Boolean   @default(false)
+  createdAt     DateTime  @default(now()) @map("created_at")
+  updatedAt     DateTime? @map("updated_at")
   name          String?
   email         String?   @unique
   emailVerified DateTime? @map("email_verified")
   username      String?   @unique
   password      String?
   image         String?
   accounts      Account[]
-  roles         UserRole[]
-  archived      Boolean   @default(false)
-  createdAt     DateTime  @default(now()) @map("created_at")
-  updatedAt     DateTime? @map("updated_at")
+  accessLevel   AccessLevel @map("access_level") @default(ALUMNI)
   accessStatus  AccessStatus  @default(PENDING) @map("access_status")
+  accessMode    AccessMode  @default(FULL_ACCESS) @map("access_mode")
```

Comment:
also why do you want to map its name to snake_case?

Reply:
If not use map, the field will be a camelCase in the db